specs/wp-media-editor-spec.js

```diff
@@ -0,0 +1,68 @@
+/** @format */
+
+import config from 'config';
+
+import LoginFlow from '../lib/flows/login-flow.js';
+
+import assert from 'assert';
+
+import * as driverManager from '../lib/driver-manager.js';
+import * as dataHelper from '../lib/data-helper';
+import NavBarComponent from '../lib/components/nav-bar-component';
+import SideBarComponent from '../lib/components/sidebar-component';
+import MediaPage from '../lib/pages/media-page';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const screenSize = driverManager.currentScreenSize();
+const host = dataHelper.getJetpackHost();
+
+let driver;
+
+before( async function() {
+	this.timeout( startBrowserTimeoutMS );
+	driver = await driverManager.startBrowser();
+} );
+
+describe( `[${ host }] Media: Edit Media (${ screenSize }) @parallel @jetpack`, function() {
+	this.timeout( mochaTimeOut );
+
+	describe( 'Edit Existing Media:', function() {
+		before( async function() {
+			const loginFlow = new LoginFlow( driver );
+			await loginFlow.login( { useFreshLogin: true } );
```

Comment:
Is there any specific reason to use `useFreshLogin`? I tried without it and spec works well.

Reply:
No specific reason, overlooked this in a copy paste, I'll take it out